src/pages/studyView/StudyViewPage.tsx

```diff
@@ -71,25 +73,33 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
 
     renderSurvivalPlot = (data: SurvivalType) => {
         return <div className={styles.survivalPlot}>
-            <SurvivalChart alteredPatientSurvivals={data.alteredGroup}
-                           unalteredPatientSurvivals={data.unalteredGroup}
-                           title={'test'}
-                           xAxisLabel="Months Survival"
-                           yAxisLabel="Overall Survival"
-                           totalCasesHeader="Number of Cases, Total"
-                           statusCasesHeader="Number of Cases, Deceased"
-                           medianMonthsHeader="Median Months Survival"
-                           yLabelTooltip="Survival estimate"
-                           xLabelWithEventTooltip="Time of death"
-                           xLabelWithoutEventTooltip="Time of last observation"
-                           showDownloadButtons={false}
-                           showTable={false}
-                           showLegend={false}
-                           styleOpts={{
-                               width: 450,
-                               height: 300
-                           }}
-                           fileName="Overall_Survival"/>
+            <div className={styles.survivalPlotTitle}>{data.title}</div>
+            <div className={styles.survivalPlotBody}>
+                <If condition={!this.store.survivalPlotData.isComplete}>
+                    <StudyViewComponentLoader></StudyViewComponentLoader>
```

Comment:
Why not use `Loader` component from `src/shared/components/loadingIndicator/LoadingIndicator`?

Reply:
They are both wrappers of TreeBounce but this one only has styles specifically applied to study view. I don't see the benefits of making LoadingIndicator accept one more property for the div and duplicate the styles every place uses it.